workspaces/templates-lib/packages/utils-aws-http-api-local/src/expressRequestToHttpApiGatewayEvent.ts

```diff
@@ -205,7 +205,11 @@ export const injectGatewayResultIntoResponse = (
         });
       });
     }
-    resp.json(JSON.parse(structuredResult.body || '"parsing error"'));
+    if (structuredResult.body) {
+      resp.json(JSON.parse(structuredResult.body || ''));
+    } else {
+      resp.end();
```

Comment:
Was not entirely sure about this one. I guess just needs a way to tell express that there is no need to wait for any body data.

Reply:
Docs aren't precise on this use case, but I think this is correct.